lib/elixir/test/elixir/kernel/raise_test.exs

```diff
@@ -236,7 +236,7 @@ defmodule Kernel.RaiseTest do
       x in [FunctionClauseError] -> Exception.message(x)
     end
 
-    assert result == "no function clause matching in Kernel.RaiseTest.zero/1"
+    assert result == "there is no definition of Kernel.RaiseTest.zero/1 that expects the arguments at positions:\n\nKernel.RaiseTest.zero(1)"
```

Comment:
I think a doctest checks only the beginning of the error message? Otherwise we will have to remove the doctest.